Gemfile

```diff
@@ -50,7 +50,7 @@ end
 
 group :release do
   gem "puppet-blacksmith", '~> 3.4',                                             require: false
-  gem "pdk",                                                                     platforms: [:ruby]
+  gem "pdk", '~> 2.0',                                                           platforms: [:ruby]
```

Comment:
for some reason, not specifying a version for pdk makes bundler download version 0.0.1 of pdk... which is bad